app/models/dynamic_annotation/annotation_type.rb

```diff
@@ -14,11 +14,8 @@ def json_schema_enabled?
   private
 
   def annotation_type_is_available
-    begin
-      self.annotation_type.camelize.constantize
+    if !self.annotation_type.blank? && Object.const_defined?(self.annotation_type.camelize)
```

Comment:
I'm a little confused by this - I'm reading it as "if we have the annotation type, and the annotation type class is defined, add an error" - is that understanding correct? If so, is that what's intended? it reads like we want to do an unless, or some other logic

Reply:
Yes, that's the intent. We have core annotation types that are defined by models in the code base. So, `Tag` and `Comment` can't be custom annotation types because there is a class for them, they are already core annotations.